resources_pages/install_ds_stack_mac.md

```diff
@@ -581,19 +648,40 @@ For more examples and ideas, visit:
  https://docs.docker.com/get-started/
 ```
 
+## Quarto CLI
```

Comment:
```suggestion
## Quarto
```

Or change the link in the ToC

Reply:
The link is pointing to Quarto CLI download, I am not sure what your re referring to

Reply:
Sorry I was unclear. I am referring to the Table of Contents. If you go here https://ubc-mds.github.io/resources_pages/install_ds_stack_mac/ and click on "quarto", you will see that it doesn't work, because it is linking to a heading named just "Quarto"